sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -248,14 +249,29 @@ internal class RadarLocationManager(
         }
     }
 
+    internal fun updateTrackingFromMeta(meta: RadarMeta?) {
+        if (meta?.remoteTrackingOptions != null) {
+            // use remotely-configured options if specified
+            logger.d("Listening to server tracking options")
```

Comment:
For this message, let's do:
`Adding server tracking options`

Reply:
same here